utils/actions/getOpenAISummary.ts

```diff
@@ -64,18 +64,28 @@ export default async function getOpenAISummary({
 
   const prompt = `Summarize what the ${summaryPrompt} ${
     commitList?.length ? `the ${commitList?.length} commits,` : ""
-  } are about. What do they tell us about the business logic? Don't summarize each piece or block of data separately, combine them and process all data. Take into consideration the current PR title and body. Don't look at each part or service of the list as a separate thing, but as a whole. The list will be available to me so you don't need to repeat it. Try to keep the summary to 3 or 4 sentences, but if it's a smaller thing just mention that.
-  Here is the list:\n  ${promptList} \n`;
+  } are about. What do they tell us about the business logic? Don't summarize each piece 
+  or block of data separately, combine them and process all data. Take into consideration 
+  the current PR title and body. Don't look at each part or service of the list as a 
+  separate thing, but as a whole. The list will be available to me so you don't need to 
+  repeat it. Here is the list:\n  ${promptList} \n`;
+
   try {
     const completion = await openai.createChatCompletion({
       model: "gpt-3.5-turbo-16k",
       messages: [
         {
           role: "system",
-          content:
-            "You are a Technical PM, that understands the business and will help the user know what is going on in this recently opened Pull Request. The user will give you some context and you will summarize it in a succinct but not jargon filled way. You will avoid going over each individual data point, but will reason about the business logic.",
+          content: `You are a Technical PM, that understands the business and will help the user 
+            know what is going on in this recently opened Pull Request. The user will give 
+            you some context and you will summarize it in a succinct but not jargon filled 
+            way. You will avoid going over each individual data point, but will reason about
+            the business logic. Be concise and don't explain step by step. Don't explain the
+            PR title in one sentence, the PR body in another one, etc. Just give a high-level
+            overview of what the PR is doing. Make it less than 45 words. ${prompt} 
+            Don't explain PR and commit message separately, merge into a single explanation.
+            Don't say "the commit message".`,
         },
-        { role: "user", content: prompt },
```

Comment:
Why did we move it to a system prompt?

Reply:
To make the code simpler to understand by having a single prompt, as we do on the labelPullRequest function

Reply:
@EstebanDalelR WDYT?